### PR TITLE
Add history, info and play icon

### PR DESCRIPTION
**Pull request by grynag**
Add history and info icon from Figma (system icons)

### CODE REVIEW

File: src/components/icons/components/action/HistoryIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+
+import useIcon, { IconProps } from '../../../../hooks/useIcon';
+
+const History = (props: IconProps) => {
+    const iconFactory = useIcon(
+        <path
+            clip-rule="evenodd"
```
Comment (MartinForre) [inline src/components/icons/components/action/HistoryIcon.tsx]: JSX props should be camel cased (clipRule, fillRule etc.)

File: src/components/icons/components/action/index.ts [hunk 2]
```diff
@@ -5,3 +5,5 @@ export { default as SortIcon } from './SortIcon';
 export { default as DropdownArrow } from './DropdownArrow';
 export { default as CollapseIcon } from './CollapseIcon';
 export { default as CalendarIcon } from './CalendarIcon';
+export { default as InfoIcon } from './InfoIcon';
+export { default as HistoryIcon } from './HistoryIcon';
```
Comment (MartinForre) [inline src/components/icons/components/action/index.ts]: Make sure to export all new icons from src/index.ts aswell

File: src/components/icons/components/action/HistoryIcon.tsx (hunk 1 above)
Comment (grynag) [reply src/components/icons/components/action/HistoryIcon.tsx]: Updated in the new commit

File: src/components/icons/components/action/index.ts (hunk 2 above)
Comment (grynag) [reply src/components/icons/components/action/index.ts]: I have exported the icons from src/index.ts in the new commit 👍

**Pull request merged**